packages/yourloops/app/app.tsx

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021-2022, Diabeloop
+ * Copyright (c) 2021-2023, Diabeloop
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏Copyright update not needed

Reply:
My IDE won't let me revert it :slightly_smiling_face: